www/src/css/snaplist.css

```diff
@@ -34,6 +44,10 @@
   margin-top:140%;
 }
 
+.b-store .b-snaplist_grid .b-snaplist__item:before {
```

Comment:
See the nested styles under `.b-snaplist_row` in this file.

The app supports switching between a row view and a grid view of the snaps. This is affected by toggling the class of the containing element between `.b-snaplist_row` and `.b-snaplist_grid`.

The design I'm working towards implementing now features only a grid of snap cards but the [ongoing visual design work](https://github.com/CanonicalLtd/snappy-design/issues/24) by @kwm14 and @jamiedawsonyoung features this toggling between gird and row hence I've been reluctant to strip out this nesting.